### DISCUSSION TITLE
Icon Request: fa-archive-out

### DISCUSSION THREAD

**Discussion by Morialkar** [FortAwesome/Font-Awesome#21228] [Icon Requests]
An icon that represents removing the action of removing an item from the archive, mainly the current archive icon without the cover on the box and with an arrow pointing out of it, a bit like the reply arrow

![oie_xwnoy7g9hflq](https://cloud.githubusercontent.com/assets/4161246/6215115/b79ea290-b5cb-11e4-82d6-908f98a216d6.png)

**Comment by jspang** [comment]
+1

Also need fa-archive-in

I suggest the arrow pointing straight up (fa-archive-out) and straight down (fa-archive-in) on the existing fa-archive icon.

Have an application with these actions in it already, so I can promise they will get some use. :-)

**Comment by brightertools** [comment]
+1 for me too, a variation would be un-archive with a minus or arrow out of the existing archive icon.

In our application we have the ability to archive, un-archive and archive ALL, so possubly some double arrow for archiving all would also help.

I think I prefer the up/down arrows too.

but certainly needs both in/out icons

**Comment by juanrmn** [comment]
+1

**Comment by santiagovazquez** [comment]
+1

**Comment by thiotes** [comment]
+1

**Comment by titivermeesch** [comment]
This would definitely be useful. There is already something similar for the trash bin, but not for the archive icon.

**Comment by hpvd** [comment]
+1

**Comment by DaniloOliveira28** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
